content/docs/tutorials/get-started/data-versioning.md

```diff
@@ -43,27 +48,26 @@ $ git commit -m "Add raw data"
 
 <details>
 
-### Expand to see what happened internally
+### 💡Expand to see what happens under the hood
```

Comment:
```suggestion
### 💡 Expand to see what happens under the hood
```
Or avoid that emoji which as of now we use for external tool hints.

Reply:
>  for external tool hints.

not sure I got this part

thanks for the fix though